modules/statistics/php/stats_demographic.class.inc

```diff
@@ -77,14 +78,14 @@ class Stats_Demographic extends \NDB_Form
         $dropdown_name,
         $dropdown_opt,
         $dropdown_selected,
-        $centres,
+        $centers,
         $data,
         $Subsection="",
         $disclamer='',
         $projectID=null
     ) {
-
-        $tpl_data = array();
+        $projectID = $projectID == '' ? null : $projectID;
```

Comment:
Small thing, but how do you feel about this instead?
`$projectID = empty($projectID) ? null : $projectID;`

I find it a bit easier on the brain.

Reply:
I'm all for it